Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.cpp

```diff
@@ -469,6 +470,7 @@ SpectralFieldDataRZ::ForwardTransform (const int lev,
                                        amrex::MultiFab const & field_mf_t, int const field_index_t)
 {
     amrex::LayoutData<amrex::Real>* cost = WarpX::getCosts(lev);
+    bool do_costs = WarpXUtilLoadBalance::doCosts(cost, field_mf_r.boxArray(), field_mf_r.DistributionMap());
```

Comment:
In RZ how do we decide whether to use `field_mf_r` or `field_mf_t` in order to get the box array and distribution mapping? Or do the radial and azimuthal MultiFabs always share the same box array and distribution mapping (maybe a question to double check with @dpgrote)?

Reply:
I think we assume they are the same (modulo the index type) below, when we use the same `mfi` to access fabs from both.